components/kyma-operator/pkg/kymaoperation/steps/install.go

```diff
@@ -38,36 +37,47 @@ func (s installStep) Run() error {
 		releaseOverrides)
 
 	if installErr != nil {
-		installErrMsg := fmt.Sprintf("Helm install error: %s", installErr.Error())
+		return s.onError(installErr)
+	}
 
-		isDeletable, err := s.helmClient.IsReleaseDeletable(s.component.GetReleaseName())
-		if err != nil {
-			statusErrMsg := fmt.Sprintf("Checking status of %s failed with an error: %s", s.component.GetReleaseName(), err.Error())
-			log.Println(statusErrMsg)
-			return errors.New(fmt.Sprintf("%s \n %s \n", installErrMsg, statusErrMsg))
-		}
+	s.helmClient.PrintRelease(installResp.Release)
 
-		if isDeletable {
+	return nil
+}
 
-			log.Println(fmt.Sprintf("Helm installation of %s failed. Deleting before retrying installation.", s.component.GetReleaseName()))
-			_, err := s.helmClient.DeleteRelease(s.component.GetReleaseName())
+func (s installStep) onError(installErr error) error {
+	installErrMsg := fmt.Sprintf("Helm install error: %s", installErr.Error())
 
-			if err != nil {
-				deleteErrMsg := fmt.Sprintf("Helm delete of %s failed with an error: %s", s.component.GetReleaseName(), err.Error())
-				return errors.New(fmt.Sprintf("%s \n %s \n", installErrMsg, deleteErrMsg))
-			}
+	isDeletable, checkErr := s.helmClient.IsReleaseDeletable(s.component.GetReleaseName())
+
+	if checkErr != nil {
+		statusErrMsg := fmt.Sprintf("Checking status of release \"%s\" failed with an error: %s", s.component.GetReleaseName(), checkErr.Error())
+		log.Println(statusErrMsg)
```

Comment:
This will lead to printing the `statusErrMsg` twice

Reply:
Done.